README.md

```diff
@@ -243,3 +243,7 @@ Copyright 2022-2023 Mina Foundation, Inc.
 
 Free use of this software is granted under the terms of the Mozilla
 Public License 2.0.
+
+## Contributing
+
+This project uses [C4(Collective Code Construction Contract)](https://rfc.zeromq.org/spec:42/C4/) process for contributions.
```

Comment:
Not sure where you got this ':' in the URL, but https://rfc.zeromq.org/spec/42/ works for me, and is redirected to from the URL that you give.

Reply:
I grabbed it from the libzmq README. I'll correct the URL